index.js

```diff
@@ -0,0 +1,84 @@
+const menuIconEl = document.getElementById('menu-icon');
+const menuIconCloseEl = document.getElementById('menu-close-icon');
+const menuEl = document.getElementById('nav-links');
+
+menuIconEl.addEventListener('click', () => {
+  menuEl.style.display = 'flex';
+  menuIconEl.style.display = 'none';
+  menuIconCloseEl.style.display = 'inline-block';
+  document.body.style.overflow = 'hidden';
+});
+
+menuIconCloseEl.addEventListener('click', () => {
+  menuEl.style.display = 'none';
+  menuIconEl.style.display = 'block';
+  menuIconCloseEl.style.display = 'none';
+  document.body.style.overflow = 'auto';
+});
+
+const mainEl = document.getElementById('main');
+const speakerSectionEl = document.createElement('section');
+speakerSectionEl.className = 'speaker-section';
+const mainSpeakerHeading = document.createElement('h2');
+mainSpeakerHeading.textContent = 'Featured Speakers';
+const mainHeadingDivider = document.createElement('div');
+mainHeadingDivider.className = 'divider';
+const speakersContainerEl = document.createElement('div');
+speakersContainerEl.className = 'speakers-container';
+
+speakerSectionEl.append(mainSpeakerHeading, mainHeadingDivider, speakersContainerEl);
+mainEl.insertBefore(speakerSectionEl, mainEl.children[2]);
+const speakerData = [
+  {
+    image: 'ab-sam.png',
+    name: 'Aboagye Sam',
+    portfolio: 'Supervising Lead, Whizzy Music.',
+    details: 'Mr. Sam is a celebrated thought leader in the field of music business. His exploits range from production to talent management',
+  },
+  {
+    image: 'reagan.png',
+    name: 'Reagan Akowuah Basil',
+    portfolio: 'C.E.O, Basil Tech Gh.',
+    details: 'Reagan is a trail blazer when it comes to emerging technologies in the music industry. He also champions the mass adoption for new tech.',
+  },
+  {
+    image: 'tetteh.jpg',
+    name: 'Kodjo-Sarso Tetteh',
+    portfolio: 'Music Director, Love Community Chapel int.',
+    details: 'Tetteh is a multi- instrumentalist who serves as the Music Director for variious choirs.',
+  },
+  {
+    image: 'jonah.jpg',
+    name: 'Mr. Jonah Frimpong Domfeh',
+    portfolio: 'Music tutor, Tritone Music Academy',
+    details: 'Mr Jonah is a tutor of music theory. His interests lie in producing Jazz music with AI.',
+  },
+  {
+    image: 'clement.jpg',
+    name: 'Mr. Clement Ofori',
+    portfolio: 'Business Analyst, Tritone Music Co',
+    details: 'Mr Clement is a music entrepreneur specializing in the commercial aspects of music production.',
+  },
+  {
+    image: 'MrDoe.jpg',
+    name: 'Mr. Alex Doe',
+    portfolio: 'Sound Engineer, Tritone Studios',
+    details: 'rising star in the feild of sound engineering.',
+  },
+];
```

Comment:
- [ ] You did a good job adding more text as requested of you by the previous reviewer. However, this currently distorts the design and on mobile view, the other description of the speaker are hidden and cannot be seen. I would suggest reducing the size of the text or ensuring that you are using a further refined description which is not so long and describes the speaker well.

**This is your design**

![image](https://user-images.githubusercontent.com/86318284/212382551-efc03b1f-16ae-49dd-824e-e92ff2f534fa.png)


**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/212382619-a9dc0172-962e-427c-881c-e5d8c751f9cb.png)

Reply:
fixed